lib/matplotlib/cbook.py

```diff
@@ -1161,10 +1162,14 @@ def boxplot_stats(X, whis=1.5, bootstrap=None, labels=None,
         Number of times the confidence intervals around the median
         should be bootstrapped (percentile method).
 
-    labels : array-like, optional
+    tick_labels : array-like, optional
         Labels for each dataset. Length must be compatible with
         dimensions of *X*.
 
+        .. versionchanged:: 3.9
+        Renamed from *labels*, which is deprecated since 3.9
+        and will be removed in 3.11.
```

Comment:
I _think_ the doc failure is because this need’s to be indented.

Reply:
It seems, you've accidentally removed the text.